src/main/java/com/example/footballTeam/users/dao/InMemoryUserDAO.java

```diff
@@ -0,0 +1,24 @@
+package com.example.footballTeam.users.dao;
+
+import com.example.footballTeam.users.model.User;
+import com.example.footballTeam.users.repository.InMemoryDatabase;
+import org.springframework.stereotype.Repository;
+
+@Repository("firstUserDAO")
+public class InMemoryUserDAO implements UserDAO {
+
+    @Override
+    public int addUser(User user) {
+        return InMemoryDatabase.addUser(user);
+    }
+
+    @Override
+    public int deleteUser(User user) {
```

Comment:
qui devi richiamare il metodo di InMemoryDatabase

Reply:
Devo creare un "deleteUserById" tipo?

Reply:
hai già il metodo in InMemoryDatabase